src/components/Bar.jsx

```diff
@@ -218,9 +220,13 @@ export class Bar extends Component {
       ? { '--cozBarThemePrimaryContrastTextColor': pctColor }
       : {}
     const themeStyle = { ...pStyle, ...pctStyle }
+    const flagshipClass = isFlagshipApp ? ' coz-is-flagship-app' : ''
 
     return (
-      <div className={`coz-bar-wrapper coz-theme-${theme}`} style={themeStyle}>
+      <div
+        className={`coz-bar-wrapper coz-theme-${theme}${flagshipClass}`}
```

Comment:
why not using classNames ?

Reply:
Well it's not in the dependencies, I didn't want to add any runtime dependency here as I wanted minimal changes